StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PollingViewModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  PollingViewModel.swift
+//  StripePaymentSheet
+//
+//  Created by Fionn Barrett on 08/08/2023.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
+@_spi(STP) import StripePaymentsUI
+@_spi(STP) import StripeUICore
+import UIKit
+
+class PollingViewModel {
+
+    let paymentMethodType: STPPaymentMethodType
+    var CTA: String
+    var deadline: Date
+
+    init(paymentMethodType: STPPaymentMethodType) {
+        self.paymentMethodType = paymentMethodType
```

Comment:
Can we assert/throw a fatal error if a non-supported payment type is injected into `PollingViewModel`?